lib/model/query/sessions.js

```diff
@@ -29,11 +29,27 @@ returning *`)
 
 const _unjoiner = unjoiner(Session, Actor);
 const getByBearerToken = (token) => ({ maybeOne }) => (isValidToken(token) ? maybeOne(sql`
-select ${_unjoiner.fields} from sessions
-join actors on actors.id=sessions."actorId"
-where token=${token} and sessions."expiresAt" > now()`)
+  SELECT "sessions"."csrf" AS "sessions!csrf"
+       , "actors"."id" AS "actors!id"
+       , "actors"."type" AS "actors!type"
+       , "actors"."acteeId" AS "actors!acteeId"
+    FROM sessions
+    JOIN actors ON actors.id=sessions."actorId"
+    WHERE token=${token}
+      AND sessions."expiresAt" > NOW()
+      AND actors."deletedAt" IS NULL
+`)
+  .then(map(auth => assoc('sessions!token', token, auth)))
```

Comment:
Are you not `SELECT`ing `sessions.token` in order to avoid a performance hit? This is another place where I feel like it'd be nice to avoid referencing `!` outside `unjoiner()` unless it's really important to do so. How much do you think it helps to avoid `SELECT sessions.token`?